### DISCUSSION TITLE
Hiding a Collection in entire Control-Panel

### DISCUSSION THREAD

**Discussion by khorramMohammadMahdi** [statamic/cms#7194] [General]
hi there ✌
i'm creating an addon and i need to store some records (might be a a lot) as flat-files, they're vulnerable and will be used only at backscene, so obviously i don't want them to be seen or modified by user.
as you know if i store them as Entries in a Collection,  that collection will be listed in Collections list and will have blueprint and will be creatable and modifiable and ...
and i will be doomed

so, is there any way that i hide a collection or create a custom repository of my own and still be able to querying, filtering, storing, saving and deleting ... in a convenient way like the one that i have with Statamic\Facades\Entry Fecade ?

**Comment by duncanmcclean** [comment]
You might actually be better creating a Stache store for whatever you're storing. Then you can store 'things' and you don't need to show them anywhere in the Control Panel.

I can't find any documentation on creating a Stache store but there's a few addons you could look at that do:

* [Redirects](https://github.com/riasvdv/statamic-redirect) (for redirects)
* [YumYum](https://github.com/duncanmcclean/yumyum) (for feeds)
* [Advanced SEO](https://github.com/aerni/statamic-advanced-seo)

Otherwise, you could potentially use permissions to limit the collections users see (however, Super Admins would still see them). You could also hide links to it with JavaScript but then it would still show in things like dropdowns in the Entries fieldtype etc.

**Reply by mgussekloo** [reply]
Hiding a collection in the control panel would be useful! Any updates on this or on the Stache Store since 2022?

**Reply by duncanmcclean** [reply]
No updates. You can either use permissions to prevent users from accessing a collection, or hide links to it from the nav using the [CP Nav Customiser](https://statamic.dev/customizing-the-cp-nav).

**Comment by tobleo24** [comment]
If anyone wants a leather jacket in good quality,  We are offering new [Arnold stylish jacket](https://tobleo.com/product/world-of-tanks-arnold-holiday-ops-brown-jacket/) for our customers.
